blosc/toplevel.py

```diff
@@ -292,6 +292,36 @@ def get_cbuffer_sizes(bytesobj):
     return _ext.get_cbuffer_sizes(bytesobj)
 
 
+def cbuffer_validate(bytesobj):
+    """
+    validate_cbuffer(bytesobj)
+
+    Validate the cbuffer. Check that the cbuffer is safe to compress. Note
+    that, this does not guarantee that the blosc will be able to decompress the
```

Comment:
I'd remove the comma after the `that`, but YMMV ;)

Reply:
fixed